README.md

```diff
@@ -79,6 +79,14 @@ The following environment variables are required in order for the app to start:
 | HMPPS_SQS_QUEUES_EDUCATIONANDWORKPLAN_QUEUE_NAME | The queue to receive events from the domain-events topic.                 |
 | HMPPS_SQS_QUEUES_EDUCATIONANDWORKPLAN_DLQ_NAME   | The dead letter queue for any failed messages.                            |
 
+### APIs
+
+| Name                   | Description                                                                                     |
+|------------------------|-------------------------------------------------------------------------------------------------|
+| CIAG_INDUCTION_API_URL | The URL of the CIAG Induction API, used as part of the ETL for the CIAG Induction API migration |
+| CIAG_API_CLIENT_ID     | hmpps-auth oauth2 client-id for connecting to the CIAG Induction API                            |
```

Comment:
The `CIAG_API_CLIENT_ID` and `CIAG_API_CLIENT_SECRET` are for the oauth client that has the necessary role to call the CIAG API. This has been setup for us by the HAAR team and provided in our k8s namespace (in `dev` only at this point.)